REQUIREMENTS.rst

```diff
@@ -0,0 +1,72 @@
+============================
+ CESM software requirements
+============================
+
+Software requirements for installing, building and running CESM
+===============================================================
+
+Installing, building and running CESM requires:
+
+* a Unix-like operating system (Linux, AIX, OS X, etc.)
+
+* git client version 1.8 or newer
+
+* subversion client version ??? or newer
+
+* python2 version 2.7 or newer
+
+* perl version ??? or newer
+
+* build tools gmake and cmake
+
+* Fortran and C compilers
+
+  * See `Details on Fortran compiler versions`_ below for more information 
+
+* LAPACK and BLAS libraries
```

Comment:
I know we need at least one of LAPACK and/or BLAS, for CLM and other components. I can never keep these straight. @jedwards4b do you know more details on what we should list here?